### ISSUE TITLE
Remove route demo

### ISSUE THREAD

**Issue by RealWeeks** [ga-wdi-boston/ember-object#11]
Seems unnecessary.

**Comment by RealWeeks** [comment]
removed in commit 14f5e3a

**Comment by gaand** [comment]
Please explain why this was in this lesson and why you removed it.  Thanks!

**Comment by RealWeeks** [comment]
@gaand No idea why it was in the lesson, so I removed it.  Seemed out of scope.

**Comment by RealWeeks** [comment]
On second thought, I think it's better removed.  This is covered very in depth in routing-static

**Comment by gaand** [comment]
Okay.  I just want to make sure that before you remove something form a lesson, you've thought about why it might be there.  For instance, does covering it briefly here support covering it in more depth later?  I don't know, but it might be worth thinking through.

**Comment by jrhorn424** [comment]
I agree with @gaand. Please take a bit longer to explain your thoughts in PRs and issues. It helps me since I'm not in the room and not always a part of the conversations you have.
